dotty-tensorflow/src/main/scala/me/shadaj/scalapy/tensorflow/api/keras/Keras.scala

```diff
@@ -5,6 +5,18 @@ import me.shadaj.scalapy.tensorflow.keras.{Keras => PyKeras}
 import me.shadaj.scalapy.tensorflow.api.scalaUtils.{PythonModule, PythonType}
 import scala.language.implicitConversions
 import optimizers.Optimizers
+import layers.Layers
+import preprocessing.Preprocessing
+import models.Models
+import datasets.Datasets
+import backend.Backend
 class Keras private[api] (val underlying: PyKeras) extends PythonType[PyKeras] with PythonModule {
```

Comment:
This comment is for whole PR as well as for other batches -- I'm strongly convinced that exposing public api without a line of comment does not make users' lives easier.

Reply:
well thats true, although documentation can be found on tensorflow website.
Anyway we had somewhat inconclusive discussion about this. Lets make an issue :D and don't worry for now.